toolchain/sem_ir/file.h

```diff
@@ -226,6 +226,14 @@ class File : public Printable<File> {
 
   auto parse_tree() const -> const Parse::Tree& { return *parse_tree_; }
 
+  auto add_non_owning_library(FunctionId function_info) {
+    non_owning_functions_.push_back(function_info);
+  }
+
+  auto get_non_owning_functions() const -> llvm::SmallVector<FunctionId, 0> {
```

Comment:
Style: We'd typically not write "get_" for accessors, per [function names](https://google.github.io/styleguide/cppguide.html#Function_Names)